### ISSUE TITLE
Inconsistent decimal places

### ISSUE THREAD

**Issue by janten** [docker/go-units#9]
go-units removes trailing zeros in its output. This makes the output in Docker's progress indicators not line up.

Current:
```
cba48ac2c991: Downloading    23 MB/37.18 MB
f19be92d17a1: Downloading  24.3 MB/64.6 MB
```

Better:
```
cba48ac2c991: Downloading  23.0 MB/37.18 MB
f19be92d17a1: Downloading  24.3 MB/64.60 MB
```

One decimal place should be enough when dealing with sizes:
```
cba48ac2c991: Downloading  23.0 MB/37.2 MB
f19be92d17a1: Downloading  24.3 MB/64.6 MB
```

**Comment by calavera** [comment]
I agree, what do you think about opening a pull request? This should be fairly simple to fix :raised_hands:

**Comment by ripcurld00d** [comment]
If I may - `fmt.Printf("%.4g\n", 10.0)` prints `10`, even though I was expecting to see `10.0`.
Reading the fmt godoc:

> %g	        %e for large exponents, %f otherwise

 gives the impression that I was correct.
So did I get the wrong idea or we need to open a ticket to the Golang team?
Either way, I would like to work on that.

Thanks.

**Comment by thaJeztah** [comment]
Oh, I think I once suggested this https://github.com/docker/docker/issues/9977

**Comment by ripcurld00d** [comment]
From the [fmt unit tests](https://github.com/golang/go/blob/master/src/fmt/fmt_test.go#L506) it looks like I got the wrong impression.

I guess we need to replace the `"%.4g"` format anyway.

If you don't mind, I will figure it out and post a PR in the next days.

**Comment by janten** [comment]
Thanks everybody for looking into this. I do currently not have the time to write a PR myself.
